tests/vector_stores/test_postgres.py

```diff
@@ -252,6 +267,21 @@ async def test_add_to_db_query_and_delete(
     assert len(res.nodes) == 1
     assert res.nodes[0].node_id == "aaa"
 
+    pg_dict = pg.to_dict()
+    await pg.close()
+
+    loaded_pg = cast(PGVectorStore, load_vector_store(pg_dict))
+
+    if use_async:
+        res = await loaded_pg.aquery(q)
+    else:
+        res = loaded_pg.query(q)
+    assert res.nodes
+    assert len(res.nodes) == 1
+    assert res.nodes[0].node_id == "aaa"
+
+    await loaded_pg.close()
+
```

Comment:
maybe add one more sanity check `assert loaded_pg.to_dict() == pg_dict`? To make sure all fields are still being loaded properly after repeated serializations/deserializations.